pkg/runtime/ast/create_table.go

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ast
+
+import (
+	"strings"
+)
+
+import (
+	"github.com/pkg/errors"
+)
+
+import (
+	"github.com/arana-db/parser/ast"
+	"github.com/arana-db/parser/format"
+)
+
+var _ Statement = (*CreateTableStmt)(nil)
+
+type CreateTableStmt struct {
+	IfNotExists bool
+	//TemporaryKeyword
+	// Meanless when TemporaryKeyword is not TemporaryGlobal.
+	// ON COMMIT DELETE ROWS => true
+	// ON COMMIT PRESERVE ROW => false
+	//OnCommitDelete bool
+	Table       *TableName
+	ReferTable  *TableName
+	Cols        []*ast.ColumnDef
+	Constraints []*ast.Constraint
+	Options     []*ast.TableOption
+	//Partition      *PartitionOptions
+	//OnDuplicate    OnDuplicateKeyHandlingType
+	//Select         ResultSetNode
+}
+
+func NewCreateTableStmt() *CreateTableStmt {
+	return &CreateTableStmt{}
+}
+
+func (c *CreateTableStmt) Restore(flag RestoreFlag, sb *strings.Builder, args *[]int) error {
+	sb.WriteString("CREATE TABLE ")
+	if c.IfNotExists {
+		sb.WriteString(" IF NOT EXISTS ")
+	}
+	if err := c.Table.Restore(flag, sb, args); err != nil {
+		return errors.Wrapf(err, "an error occurred while restore AnalyzeTableStatement.Tables[%s]", c.Table)
```

Comment:
Could you unify the same capitalization of error messages in the project?

Reply:
OK